pkg/utils/utils.go

```diff
@@ -27,13 +27,13 @@ var onComponents = []string{
 	backplanev1.ClusterProxyAddon,
 	backplanev1.LocalCluster,
 	backplanev1.HypershiftLocalHosting,
+	backplanev1.HyperShift,
 	// backplanev1.ConsoleMCE, // determined by OCP version
-	// backplanev1.HyperShift,
 }
 
 var offComponents = []string{
 	backplanev1.ManagedServiceAccount,
-	backplanev1.HyperShift,
+	backplanev1.HyperShiftPreview,
```

Comment:
I think this will lead the preview component to be added to the list of components on new installs, which we don't want. If you move it to the list of all components though I think it will stop populating it in the spec, and still not complain if it's already in the spec.

Reply:
Changes were made. Resolving this request.